sphinx/apidoc.py

```diff
@@ -404,6 +410,10 @@ def main(argv=sys.argv):
             module_path = rootpath,
             append_syspath = opts.append_syspath,
         )
+        enabled_exts = {'ext_'+ext: getattr(opts, 'ext_'+ext)
```

Comment:
Could you add spaces before and after `+` signs?
I will enable the flake8 checks for the spaces.